pkg/cmd/cli/cmd/rollback.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kubectl "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	kcmd "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd"
+
+	describe "github.com/openshift/origin/pkg/cmd/cli/describe"
+	deployapi "github.com/openshift/origin/pkg/deploy/api"
+)
+
+const rollbackLongDesc = `
+Revert part of an application back to a previous deployment.
+
+When you run this command your deployment configuration will be updated to match
+the provided deployment. By default only the pod and container configuration
+will be changed and scaling or trigger settings will be left as-is. Note that
+environment variables and volumes are included in rollbacks, so if you've
+recently updated security credentials in your environment your previous
+deployment may not have the correct values.
+
+If you would like to review the outcome of the rollback, pass '--dry-run' to print
+a human-readable representation of the updated deployment configuration instead of
+executing the rollback. This is useful if you're not quite sure what the outcome
+will be.
+
+Examples:
+  $ %[1]s %[2]s deployment-2
+  ...
+
+  $ %[1]s %[2]s deployment-3 --include-strategy=true
```

Comment:
What are the ... here for?  Describing what they do?  I don't know if --include-strategy is worth including.  Just the other three.  Go ahead and add the descriptions, this is looking good.